infrastructure/zk/src/docker.ts

```diff
@@ -11,7 +11,7 @@ const IMAGES = [
     'keybase',
     'ci',
     'exit-tool',
-    'dev-liquidity-token-watcher',
+    'zk-environment',
```

Comment:
why do we add this here?

Reply:
oh I think that must have come from the rebase, I'll check. I certainly did not add it myself (that I know of ofc).

Reply:
well, it does look like I added that accidentally, as there is no such line in either main or coingecko_2 branch. I'll remove it. I'm not sure how it got its way in tbh

Reply:
done